src/git_changelog/cli.py

```diff
@@ -487,6 +521,7 @@ def build_and_render(
     bump: str | None = None,
     zerover: bool = True,  # noqa: FBT001,FBT002
     filter_commits: str | None = None,
+    jinja_context: dict[str, str] | None = None,
```

Comment:
It's true that values passed from the CLI will always be strings, but when using the API directly from Python, values could be anything, so I think it should be `dict[str, Any]`.